e2e/app/page/notebook-page.ts

```diff
@@ -45,7 +45,13 @@ export default class NotebookPage extends AuthenticatedPage {
 
   async isLoaded(): Promise<boolean> {
     await waitForDocumentTitle(this.page, this.documentTitle);
-    await this.waitForKernelIdle(30000);
+    try {
+      await this.findRunButton(120000);
+    } catch (err) {
+      console.log(`Reload "${this.documentTitle}" because cannot find the Run button`);
```

Comment:
nit: `Reload*ing*` seems a little more accurate since it's going to execute the action itself (and it's not asking the user to do so).

Reply:
right, done!